docs/design/autoware-interfaces/ad-api/list/api/planning/index.md

```diff
@@ -0,0 +1,96 @@
+# Planning API
+
+- {{ link_ad_api('/api/planning/allow_start') }}
+- {{ link_ad_api('/api/planning/state') }}
+- {{ link_ad_api('/api/planning/factors') }}
+
+## Description
+
+This API manages the behavior that the vehicle plans.
+Applications can notify the vehicle behavior to the people around and visualize it for operator and passengers.
+
+## States
+
+The planning state manages the stop and start of the vehicle.
+Once the vehicle has stopped, the state will be STOPPED.
+After this, when the vehicle tries to start (is still stopped), the state will be STARTING.
+In this state, calling the start API changes the state to MOVING and the vehicle starts.
+This mechanism can add processing such as announcements before the vehicle starts.
+Depending on the configuration, the state may transition directly from STOPPED to MOVING.
+
+![planning-state](./docs/state.drawio.svg)
+
+| State    | Description                                     |
+| -------- | ----------------------------------------------- |
+| STOPPED  | The vehicle is stopped.                         |
+| STARTING | The vehicle is stopped, but is trying to start. |
+| MOVING   | The vehicle is moving.                          |
+
+## Factors
+
+The planning factors is an array of information on the behavior that the vehicle plans.
+Each factor has type shown below, pose, distance from the vehicle head to that pose, status, and detailed data depending on its type.
+See the description of each behavior below for pose and status. And see the description of factor types for detailed data.
+
+![planning-factors](./docs/factors.drawio.svg)
+
+| Factor Type                 | Behavior         | Description                                            |
+| --------------------------- | ---------------- | ------------------------------------------------------ |
+| SURROUND_OBSTACLE           | immediate stop   | There are obstacles nearby.                            |
```

Comment:
What is the difference between a `SURROUND_OBSTACLE` and a `FRONT_OBSTACLE`? `surround` means all around, and so this word also covers obstacles in front and behind.

Reply:
cc: @kenji-miyake 
The front has a long range because it detects on the route. So is the `ROUTE_OBSTACLE` better?

Reply:
I think this is a good name. The description can also be updated to say something like `There are obstacles along the route ahead` to make it clear that the obstacle is further ahead of the vehicle along its planned route. 

Then the description for `SURROUND_OBSTACLE` (`SURROUNDING_OBSTACLE` may be better 🤔 ) can be updated to say `There are obstacles immediately around the vehicle` (assuming this is a close-range check).

Reply:
Done https://github.com/autowarefoundation/autoware-documentation/pull/154/commits/70b0c863477c900ab713e0c702087186e96c0f77.